### DISCUSSION TITLE
Git push does't trigger a new deployment

### DISCUSSION THREAD

**Discussion by Kailash8799** [vercel/community#4259] [Help] [answered]
### Summary

 Git push does't  trigger a new deployment when i make commit, it problem happed today, it work fine more than 20 days

### Steps to Reproduce

Git push

**Comment by amyegan** [comment, marked as answer]
Do you have [Ignored Build Step](https://vercel.com/docs/projects/overview#ignored-build-step) or [git.deploymentEnabled](https://vercel.com/docs/projects/project-configuration/git-configuration#git.deploymentenabled) configured for your project? They could block certain deployments.

Another thing to check is the current [Limits](https://vercel.com/docs/limits/overview#) for your plan type. There is a limit on the number of deployments that can be created per day, for example, which could temporarily prevent new deployments.
